ShopperInsights/src/test/java/com/braintreepayments/api/ShopperInsightsClientUnitTest.kt

```diff
@@ -35,18 +43,32 @@ class ShopperInsightsClientUnitTest {
     @Test
     fun testGetRecommendedPaymentMethods_returnsDefaultRecommendations() {
         val request = ShopperInsightsRequest("fake-email", null)
-        sut.getRecommendedPaymentMethods(request) { result ->
+        sut.getRecommendedPaymentMethods(context, request) { result ->
             assertNotNull(result)
             val successResult = assertIs<ShopperInsightsResult.Success>(result)
             assertNotNull(successResult.response.isPayPalRecommended)
             assertNotNull(successResult.response.isVenmoRecommended)
         }
     }
 
+    @Test
+    fun testGetRecommendedPaymentMethods_hasBothAppsInstalled_returnsSuccessResult() {
```

Comment:
Can we add a test for the negative case here too?

Reply:
I've updated the other tests to test the negative cases.